cg/constants/tb.py

```diff
@@ -7,3 +10,12 @@ class AnalysisStatus:
     RUNNING: str = "running"
     TIMEOUT: str = "timeout"
     QC: str = "qc"
+
+
+class AnalysisTypes(str, Enum):
```

Comment:
can this be done?
```suggestion
class AnalysisTypes(StrEnum):
```

Reply:
many cases like this in cg

Reply:
I would rather remove `StrEnum` and keep `str, Enum`. StrEnum is a custom add on defined by us and I think it can be fully replaced by basing the Enum on a sting class as in `str, Enum`.

Reply:
that would be nice!